client/index.jsx

```diff
@@ -1,18 +1,62 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Container } from './components';
+import styled from 'styled-components'
+import GlobalStyle from './globalStyle';
+
+const Title = styled.div`
+  color: #fff;
+  font-size: 26px;
+  font-weight: 500;
+  letter-spacing: -0.14px;
+  line-height: 30px;
+  border-bottom: 1px solid #0e0d0d;
+  padding-bottom: 16px;
+`;
+
+const Body = styled.div`
+  display: flex;
+`
+const Inner = styled.div`
+  width: 750px;
+  margin: 0 auto;
+`
 
 class App extends React.Component {
   constructor() {
     super()
+
+    this.state = {
+      data: []
+    }
   }
 
+  componentDidMount() {
+    fetch('/getData?company=MMM')
+      .then((response) => {
+        return response.json()
+      })
+      .then((body) => {
+        this.setState({
+          data: body
+        })
+      })
+      .catch((err) => {
```

Comment:
I just remembered eric saying that with error handling, that console.logging isn't the most informative. Maybe think on the client side perspective if they make a request for company data and there's an error, a warning could show up on the screen saying 'no company data found' or something like that.